src/components/com_tjcertificate/administrator/libraries/certificate.php

```diff
@@ -432,21 +488,23 @@ public function bind(&$array)
 	/**
 	 * Method to get issued certificate list
 	 *
-	 * @param   string   $client    Client e.g. com_tjlms.course
+	 * @param   string   $client          Client e.g. com_tjlms.course
 	 *
-	 * @param   integer  $clientId  Specific client id
+	 * @param   integer  $clientId        Specific client id
 	 *
-	 * @param   integer  $userId    User Id
-	 *
-	 * @param   boolean  $expired   Get expired certificates
+	 * @param   integer  $userId          User Id
 	 *
+	 * @param   boolean  $expired         Get expired certificates
+	 * 
+	 * @param   boolean  $clientIssuedTo  Client issued to
+	 * 
 	 * @return  boolean|array Issued certificate array
 	 *
 	 * @since 1.0
 	 */
-	public static function getIssued($client, $clientId, $userId, $expired = false)
+	public static function getIssued($client, $clientId, $userId, $expired = false, $clientIssuedTo = null)
```

Comment:
Set a default value `0` to `userId` & `clientIssuedTo` 
 `public static function getIssued($client, $clientId, $userId = 0, $expired = false, $clientIssuedTo = 0)`